index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <main id="main">
+      <div class="nav">
+        <h1 id="title">Mario Vargas Llosa</h1>
+        <p class="para">The Nobel Prize of Peru</p>
+      </div>
```

Comment:
Please use semantic HTML tags instead of divs.
This helps screen readers, browsers, and other devs to better understand your page structure 🙌